docs/aqp_aws.md

```diff
@@ -0,0 +1,235 @@
+For example, for research-based companies (like Gallup), for political polls results, a small sample is used to estimate support for a candidate within a small margin of error.
+In this document, we describe the features provided by SnappyData for analysing your data. It also provides details for deploying SnappyData cluster on AWS CloudFormation or on AWS using the EC2 script. 
+Refer to the the examples and guidelines provided in this document to help you create notebooks using which, you can execute SQL queries or data frame API to analyse your data.
+##Key Components##
+This section provides a brief description of the key terms used in this document. 
+
+* **Amazon Web Services (AWS**): Amazon Web Services (AWS) is a comprehensive, evolving cloud computing platform that offers a suite of cloud-computing services. The services provided by this platform that are important for SnappyData are, Amazon Elastic Compute Cloud (EC2) and Amazon Simple Storage Service (S3).
+* **SnappyData Cluster**: A database cluster which has three main components - Locator, Server and Lead
+* **Apache Zeppelin**: Apache Zeppelin is a web-based notebook that enables interactive data analytics. It allows you to make data-driven, interactive and collaborative documents with SQL queries or directly use the Spark API to process data
+* **Interpreters**: A software module which is loaded into Apache Zeppelin upon startup. Interpreters allow various third party products including SnappyData to interact with Apache Zeppelin. The SnappyData interpreter gives users the ability to execute SQL queries or use the data frame API to visualize data.
+
+#Quick Start Steps#
+Do want to understand the product and start using it in a few minutes? Follow these easy steps that can get your started quickly:
+1. Setting up SnappyData Cluster
+        - Deploying the Cluster with AWS CloudFormation
+        - Deploying the Cluster with AWS using Script
+2. Loading Data from AWS S3
+3. Logging into Apache Zeppelin
+        - Using Predefined Notebook
+        -Creating your own Notebook
+
```

Comment:
We should also add two more steps under quick start steps. 
1. Launching Pulse Web Application
2. Launching Spark UI